ext/PlottingExt/PlotSpins.jl

```diff
@@ -128,39 +128,38 @@ function Sunny.plot_spins!(ax, sys::System; notifier=Makie.Observable(nothing),
         # These observables will be reanimated upon calling `notify(notifier)`.
         vecs = Makie.Observable(Makie.Vec3f[])
         pts = Makie.Observable(Makie.Point3f[])
-        pts_shifted = Makie.Observable(Makie.Point3f[])
-        arrowcolor = Makie.Observable(Makie.RGBAf[])
+        tipcolor = Makie.Observable(Makie.RGBAf[])
 
         Makie.on(notifier, update=true) do _
-            empty!.((vecs[], pts[], pts_shifted[], arrowcolor[]))
+            empty!.((vecs[], pts[], tipcolor[]))
 
             # Dynamically adapt `rgba_colors` according to `colorfn`
             if !isnothing(colorfn)
                 numeric_colors .= colorfn.(CartesianIndices(sys.dipoles))
                 dyncolorrange = @something colorrange extrema(numeric_colors)
                 numbers_to_colors!(rgba_colors, numeric_colors, cmap_with_alpha, dyncolorrange)
             end
-            
+
             for (site, n) in zip(idxs, offsets)
                 v = (lengthscale / s0) * vec(sys.dipoles[site])
                 pt = supervecs * (rs[site] + n)
-                pt_shifted = pt - arrow_fractional_shift * v
                 push!(vecs[], Makie.Vec3f(v))
                 push!(pts[], Makie.Point3f(pt))
-                push!(pts_shifted[], Makie.Point3f(pt_shifted))
-                push!(arrowcolor[], rgba_colors[site])
+                push!(tipcolor[], rgba_colors[site])
             end
             # Trigger Makie redraw
-            notify.((vecs, pts, pts_shifted, arrowcolor))
+            notify.((vecs, pts, tipcolor))
             # isnothing(color) || notify(arrowcolor)
         end
 
         # Draw arrows
-        linecolor = (stemcolor, alpha)
-        Makie.arrows!(ax, pts_shifted, vecs; arrowsize, linewidth, linecolor, arrowcolor, diffuse=1.15, transparency=isghost)
+        shaftcolor = (stemcolor, alpha)
+        if !isempty(pts[])
+            Makie.arrows3d!(ax, pts, vecs; align=0.37, markerscale=1, tipradius, shaftradius, tiplength, tipcolor, shaftcolor, transparency=isghost) # FIXME: diffuse=1.15
```

Comment:
If `align = 0.5`, then the arrows would be exactly centered on `pts`. A value `align < 0.5` will shift the arrows in the direction of `dirs`. The choice `align = 0.37` was empirically determined by visual comparison with previous Sunny calculations.